src/lib/eslint-naming.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * This is a direct copy and paste from eslint's github repository
+ *
+ * This will need updating if eslint changes their resolution function
```

Comment:
How does this change if the user isn’t using eslint 6?

Reply:
I'm fairly certain all changes are backwards compatible and only fixed bugs. As far as I know, if a change isn't compatible with an old version of eslint, the plugin wouldn't be compatible either.